specs/AppNotifications/AppNotificationBuider/Headers-spec.md

```diff
@@ -0,0 +1,67 @@
+# Headers
+
+Headers let developers identify related notifications so they can be grouped together in Notification Center.
+![AppNotification Header Example](toast-content-headers.png)
+
+Developers can specify a Header when defining a notification in raw xml, but the AppNotificationBuilder doesn't support it.
+
+```xml
+<toast>
+    <header 
+        id='6289' 
+        title='Camping!!' 
+        arguments='action=openConversation;id=6289'/>
```

Comment:
nit: maybe rename id as actionId=6289? At a first glance, I got confused between this id and the header id.

Reply:
This was copied straith from our docs, hope expternal devs don;t get confused ;)

but I can rename if it's a problem.